<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InsertNodeOrder</class>
 <widget class="QDialog" name="InsertNodeOrder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1098</width>
    <height>683</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Order</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <layout class="QGridLayout" name="gridLayout">
      <property name="verticalSpacing">
       <number>-1</number>
      </property>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="rBtnCash">
          <property name="text">
           <string>Cash</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rBtnMonthly">
          <property name="text">
           <string>Monthly</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rBtnPending">
          <property name="text">
           <string>Pending</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chkBoxRefund">
          <property name="text">
           <string>Refund</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QDateEdit" name="dateEdit">
          <property name="frame">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="correctionMode">
           <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
          </property>
          <property name="showGroupSeparator" stdset="0">
           <bool>false</bool>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
          <property name="currentSectionIndex">
           <number>0</number>
          </property>
          <property name="timeSpec">
           <enum>Qt::LocalTime</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1" rowspan="4">
       <widget class="QDoubleSpinBox" name="dSpinFinalTotal">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0" rowspan="3">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="lineEditOrder">
          <property name="placeholderText">
           <string>Description</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chkBoxBranch">
          <property name="text">
           <string>&amp;Branch</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Discount</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="dSpinDiscount">
          <property name="readOnly">
           <bool>false</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Company</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboCompany">
          <property name="editable">
           <bool>true</bool>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pBtnInsertStakeholder">
          <property name="text">
           <string>InsertStakeholder</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>5</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Employee</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboEmployee">
          <property name="editable">
           <bool>true</bool>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pBtnPrint">
        <property name="text">
         <string>Print</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTableView" name="tableViewOrder"/>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="pBtnSaveOrder">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pBtnCancelOrder">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pBtnOkOrder">
       <property name="text">
        <string>Ok</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tableViewOrder</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pBtnOkOrder</sender>
   <signal>clicked()</signal>
   <receiver>InsertNodeOrder</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1060</x>
     <y>664</y>
    </hint>
    <hint type="destinationlabel">
     <x>921</x>
     <y>654</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pBtnCancelOrder</sender>
   <signal>clicked()</signal>
   <receiver>InsertNodeOrder</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>662</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>661</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pBtnSaveOrder</sender>
   <signal>clicked()</signal>
   <receiver>InsertNodeOrder</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>987</x>
     <y>665</y>
    </hint>
    <hint type="destinationlabel">
     <x>800</x>
     <y>650</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
